src/components/NewEmailButton/index.jsx

```diff
@@ -0,0 +1,9 @@
+function NewEmailButton (){
```

Comment:
See my comment on your Dropdown component - this NewEmailButton should be replaced with a single re-usable component that can be used for all nav links